packages/typespec-vscode/rollup.config.ts

```diff
@@ -45,7 +46,7 @@ export default defineConfig([
     plugins: [
       ...plugins,
       ts("dist/src"),
-      copy({
+      (copy as any)({
```

Comment:
was this one necessary? seems like it was working before?

Reply:
ah make sense, thought the rollup file also was being validated in the build but apparently not, I filed a separate issue to see if we can stop using rollup as its quite a pain